cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -47,15 +47,22 @@ import           Ouroboros.Network.BlockFetch.ClientState (TraceFetchClientState
 import qualified Ouroboros.Network.BlockFetch.ClientState as BlockFetch
 import           Ouroboros.Network.BlockFetch.Decision (FetchDecision, FetchDecline (..))
 import           Ouroboros.Network.ConnectionId (ConnectionId (..))
-import           Ouroboros.Network.ConnectionManager.Types (AbstractState (..)
-                     , ConnectionManagerTrace (..), ConnectionManagerCounters (..))
+import Ouroboros.Network.ConnectionManager.Types
+    ( AbstractState(..),
+      ConnectionManagerTrace(..),
+      ConnectionManagerCounters(..),
+      AbstractState(..),
+      ConnectionManagerCounters(..),
+      OperationResult(..) )
```

Comment:
Let's preserve the way imports are written in this repository.

Reply:
oops! Thank you!